Sources/Public/Controls/AnimatedSwitch.swift

```diff
@@ -176,21 +218,27 @@ open class AnimatedSwitch: AnimatedControl {
 
   // MARK: Fileprivate
 
-  fileprivate var onStartProgress: CGFloat = 0
-  fileprivate var onEndProgress: CGFloat = 1
-  fileprivate var offStartProgress: CGFloat = 1
-  fileprivate var offEndProgress: CGFloat = 0
-  fileprivate var _isOn = false
   fileprivate var hapticGenerator: ImpactGenerator
 
+  fileprivate var _isOn = false {
+    didSet {
+      stateUpdated?(_isOn)
+    }
+  }
+
   // MARK: Private
 
   private func updateAccessibilityLabel() {
-    accessibilityValue = _isOn ? NSLocalizedString("On", comment: "On") : NSLocalizedString("Off", comment: "Off")
+    let value = _isOn ? NSLocalizedString("On", comment: "On") : NSLocalizedString("Off", comment: "Off")
```

Comment:
Do we provide localizations for those strings? If not, should we inject the bundle so that someone can provide their localizations?

Reply:
Not sure, this was here before. I think we probably don't want to change this without really understanding how changes could affect consumers.